data/human/pirate jobs.txt

```diff
@@ -3396,3 +3396,21 @@ mission "Republic Assassination [4]"
 		"reputation: Republic" -= 25
 		"reputation: Navy (Oathkeeper)" -= 25
 		"reputation: Pirate" += 10
+
+mission "Slave revolt"
+	landing
+	repeat
+	to offer
+		random < 10
+	on offer
+		require "Slave" 2
+		conversation
+			`	You are flicking through your diary in your cabin, when you hear your door start to open.`
+			`	You look up, one of your slaves is standing in the doorway... holding a large metal bar.`
+			choice
+				`    Grab for a weapon.`
+					goto grab
+			label "grab"
+			`	You immediately try to grab the antique sword hanging on the wall that your grandfather gave you before you left to be a space pilot.`
```

Comment:
Kinda dumb tbh, multiple cases(like laser rifle description) hinted that captains and crews already have sidearms, since we're in year 3000 not 300 it is much likelier the "sidearm" refers to a handgun/pistol of some sort.

Reply:
Let's not forget the fact that you literally have to take a loan to get off the planet in your own ship, or the fact that your character's past is a blank slate, minus the single point about being a factory worker.

Reply:
Worth noting, too, that multiple starting points and varied backgrounds is on the wishlist. I did PR alternate starts that kept the player being textile related, but that got shot down.